packages/core/core-flows/src/order/utils/order-validation.ts

```diff
@@ -30,3 +30,37 @@ export function throwIfItemsDoesNotExistsInOrder({
     )
   }
 }
+
+export function throwIfReturnIsCancelled({
+  orderReturn,
+}: {
+  orderReturn: ReturnDTO
+}) {
+  if (orderReturn.canceled_at) {
+    throw new MedusaError(
+      MedusaError.Types.INVALID_DATA,
+      `return with id ${orderReturn.id} has been canceled.`
+    )
+  }
+}
+
+export function throwIfItemsDoesNotExistsInReturn({
```

Comment:
```suggestion
export function validateItemExists({
```

Should suffice, you are passing an `orderReturn` input anyway, and throwing is a common way to handle invalid data anyway. Alternatively `validateItemExistsInReturn` is also fine, but I think there is no need to have the `throwIf` part